cocos/audio/audio-clip.ts

```diff
@@ -44,160 +45,76 @@ export interface AudioMeta {
  */
 @ccclass('cc.AudioClip')
 export class AudioClip extends Asset {
-    public static AudioType = AudioType;
```

Comment:
Why to remove the public class property `AudioType`?

Reply:
added.